website/src/pages/en/subgraphs/querying/graphql-api.mdx

```diff
@@ -327,24 +339,24 @@ This query will return `Challenge` entities, and their associated `Application`
 
 This query will return `Challenge` entities, and their associated `Application` entities, as they existed directly after processing the block with the given hash.
 
-### Fulltext Search Queries
+### Full-text Search Queries
 
-Fulltext search query fields provide an expressive text search API that can be added to the Subgraph schema and customized. Refer to [Defining Fulltext Search Fields](/developing/creating-a-subgraph/#defining-fulltext-search-fields) to add fulltext search to your Subgraph.
+Full-text search query fields provide an expressive text search API that can be added to the Subgraph schema and customized. Refer to [Defining Full-text Search Fields](/developing/creating-a-subgraph/#defining-fulltext-search-fields) to add full-text search to your Subgraph.
 
-Fulltext search queries have one required field, `text`, for supplying search terms. Several special fulltext operators are available to be used in this `text` search field.
+Full-text search queries have one required field, `text`, for supplying search terms. Several special full-text operators are available to be used in this `text` search field.
 
-Fulltext search operators:
+Full-text search operators:
 
 | Symbol | Operator | Description |
-| --- | --- | --- |
+| --- | --- | --- | --- |
 | `&` | `And` | For combining multiple search terms into a filter for entities that include all of the provided terms |
-| &#x7c; | `Or` | Queries with multiple search terms separated by the or operator will return all entities with a match from any of the provided terms |
+|  |  | `Or` | Queries with multiple search terms separated by the or operator will return all entities with a match from any of the provided terms |
```

Comment:
Should be `|`, no?

Reply:
I mean with the backticks. Otherwise it will be interpreted as a new column by the Markdown parser, right?

Reply:
@MichaelMacaulay (in case you don't get a notification if I don't tag you, since this PR was already merged :P)